docker/hvd/Dockerfile.hvd-apex

```diff
@@ -0,0 +1,61 @@
+# Multi-stage build
+# 1/Building apex and hvd with pytorch:1.6.0-cuda10.1-cudnn7-devel
+FROM pytorch/pytorch:1.6.0-cuda10.1-cudnn7-devel AS apex-hvd-builder
+
+ARG ARG_TORCH_CUDA_ARCH_LIST="6.0;6.1;6.2;7.0;7.5"
+ENV TORCH_CUDA_ARCH_LIST=$ARG_TORCH_CUDA_ARCH_LIST
+
+# Install git
+RUN apt-get update && apt-get install -y --no-install-recommends git
+
+# Build apex
+RUN echo "Setup NVIDIA Apex" && \
+    tmp_apex_path="/tmp/apex" && \
+    rm -rf $tmp_apex_path && \
+    git clone https://github.com/NVIDIA/apex $tmp_apex_path && \
+    cd $tmp_apex_path && \
+    pip wheel --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" .
+
+# Build Horovod
+RUN apt-get update && apt-get install -y git && \
+        git clone --recursive https://github.com/horovod/horovod.git /horovod && \
```

Comment:
Just checked Horovod repository and they have made a release 9 days ago : https://github.com/horovod/horovod/releases/tag/v0.20.0 => we have to install the stable release instead of the master

Reply:
yes I agree I will update horovod accordingly

Reply:
yes I will update horovod accordingly